modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/buttons/RowTextButton.kt

```diff
@@ -0,0 +1,48 @@
+package au.com.shiftyjelly.pocketcasts.compose.buttons
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.ButtonColors
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import au.com.shiftyjelly.pocketcasts.compose.AppThemeWithBackground
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+
+@Composable
+fun RowTextButton(
+    text: String,
+    modifier: Modifier = Modifier,
+    includePadding: Boolean = true,
+    colors: ButtonColors = ButtonDefaults.outlinedButtonColors(),
+    textIcon: Painter? = null,
+    tintIcon: Boolean = true,
+    onClick: () -> Unit,
+) {
+    RowOutlinedButton(
+        text = text,
+        modifier = modifier,
+        includePadding = includePadding,
+        border = null,
+        colors = colors,
+        textIcon = textIcon,
+        tintIcon = tintIcon,
+        onClick = onClick
+    )
+}
+
+@Preview(showBackground = true)
+@Composable
+fun RowTextButtonPreview(@PreviewParameter(ThemePreviewParameterProvider::class) themeType: Theme.ThemeType) {
```

Comment:
I kept it as an outlined button without a border to keep the same click area and highlight. The padding between the sign up and log in button is less as with the border it looks too far away.

<img width="356" alt="Screenshot 2022-11-15 at 10 56 36 am" src="https://user-images.githubusercontent.com/308331/201796205-e83df8b0-6cba-465c-a39c-17a0dfbb310c.png">

<img width="375" alt="Screenshot 2022-11-15 at 10 57 37 am" src="https://user-images.githubusercontent.com/308331/201796341-6777ebac-3f73-41b8-9842-adca3ddecc8a.png">